db/schema.rb

```diff
@@ -2482,6 +2482,7 @@
     t.datetime "confirmation_sent_at"
     t.boolean  "require_portal_user_type",                :default => false
     t.string   "sign_up_path"
+    t.boolean  "email_subscriber",                        :default => false
```

Comment:
I'm probably confused, but would a new migration be in order since this is a change to the schema?

Reply:
@DALove1025 Do you mean new as in an addition to this migration? https://github.com/concord-consortium/rigse/pull/509/commits/83c2ef5bb0d2cf1931017782334f8bc1d476d0b0

If so, would that just be if we change email_subscriber to email_subscribed (which is fine with me by the way)? Or is there another reason another migration is necessary?

Reply:
Nope, I just didn't find the migration when I went looking for it; but there it is! All good.

And of course, as you say, it would change if the attribute name changes, but only if you thought that enhanced the code.